moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -87,6 +87,7 @@ const DEFAULT_HIDDEN_COLS = {
   interim_project_id: true,
   children_project_ids: true,
   parent_project_id: true,
+  components: true,
```

Comment:
defaulting to hidden

Reply:
there is still a tiny bug with the hidden cols. if i delete my local storage data, this works as expected. but if you already have a column config in local storage, components will appear in the project list. 

the problem is that the `components` property is not in the stored column config, and so [this test](https://github.com/cityofaustin/atd-moped/blob/de594a8906024e938341f1b87caa74ec3bfbbc45/moped-editor/src/views/projects/projectsListView/helpers.js#L137) is undefined/false:

maybe we could merge the default config with the stored config [here](https://github.com/cityofaustin/atd-moped/blob/de594a8906024e938341f1b87caa74ec3bfbbc45/moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js#L238C1-L244C1), so any new columns are brought in. kinda like this?

```javascript
setHiddenColumns({...DEFAULT_HIDDEN_COLS, ...storedConfig });
````

Reply:
thank you! added the merge